front/app/containers/IdeasShow/components/TranslateButton.tsx

```diff
@@ -10,16 +10,20 @@ import { trackEventByName } from 'utils/analytics';
 import { isNilOrError } from 'utils/helperUtils';
 
 import tracks from '../tracks';
+import { getLanguage } from 'utils/i18n';
+import { SupportedLocale } from 'typings';
 
 interface Props {
   idea: IIdea;
   translateButtonClicked: boolean;
   onClick: (clicked: boolean) => void;
 }
 
+const supportedLanguages = ['ar', 'ca', 'da', 'de', 'el', 'en', 'es', 'fi', 'fr', 'hr', 'hu', 'it', 'lb', 'lv', 'mi', 'nb', 'nl', 'pl', 'pt', 'ro', 'sr', 'sv', 'tr'];
+
 const TranslateButton = ({ idea, translateButtonClicked, onClick }: Props) => {
```

Comment:
It looks like there are actually a number of Translate Buttons in our application, so the solution would have to be something more re-usable (E.g. Have `supportedLanguages` in a util file somewhere and reuse it) and also be applied to all of them. At the moment, this fix only applies to the Ideas translate button.

![Screenshot from 2024-04-15 15-19-06](https://github.com/CitizenLabDotCo/citizenlab/assets/33987955/79b63491-64ce-4088-9b46-7911782a3157)

Reply:
@sebastienhoorens I can try and take a look during my 10% time on Friday if you want? :) Not sure if we should spend time on it at the moment though with all the GoVocal work :thinking:

Reply:
Oh no, I thought this would be the same button for all 😅

Machine translations are completely broken so I think it makes sense do treat this (and the other ticket) as a serious bug. But also fine for me to look at it during 10% if you're okay with this.

@amanda-anderson Perhaps I can undo my commits and create a different branch for that part?

Reply:
> Perhaps I can undo my commits and create a different branch for that part?

Sure sounds good!! If you want to look into it during your 10% and have questions feel free to reach out as well! :)

Reply:
> Machine translations are completely broken 

Do you mean all machine translations right now? Or just for those languages which aren't supported?? If it's all of them I wasn't aware :grimacing: Indeed should be fixed in that case. Do we have a dispatch ticket for that already?

Reply:
I added a ticket here https://www.notion.so/citizenlab/Machine-translations-are-broken-f0203fb9f6cb47e0bbfb275b782ad4f1?pvs=4

I tested on a few platforms (for example https://www.zesumme-vereinfachen.lu/en-GB/ideas/cipn-comite-interministeriel-des-politiques-numeriques) and the machine translations never worked for me. It seems we have this issue since a month more or less.

Reply:
Yikes! :grimacing: Ok - yeah let's discuss in this week's planning then :) Maybe we can prioritize it for next week :crossed_fingers: